src/test/java/seedu/address/model/person/NameContainsKeywordsPredicateTest.java

```diff
@@ -70,6 +70,6 @@ public void test_nameDoesNotContainKeywords_returnsFalse() {
         // Keywords match phone, email and address, but does not match name
         predicate = new NameContainsKeywordsPredicate(Arrays.asList("12345", "alice@email.com", "Main", "Street"));
         assertFalse(predicate.test(new PersonBuilder().withName("Alice").withPhone("12345")
-                .withEmail("alice@email.com").withAddress("Main Street").build()));
+                .withEmail("alice@email.com").build()));
```

Comment:
Shouldn't there be a method call `withMatriculationNumber(...)`?

Reply:
Yes there should be. Missed it out.